ui/js/graphs.js

```diff
@@ -84,6 +84,12 @@ perf.controller('GraphsCtrl', [
                                      function(resultSetId) {
                                        return (resultSetId < dataPoint.resultSetId);
                                      });
+        var triggerCounter = _.countBy(phSeries.flotSeries.resultSetData,
+                                      function(resultSetId) {
+                                          return resultSetId === dataPoint.resultSetId ? 'trigger':'untrigger';
```

Comment:
Also here, wondering perhaps about `'retrigger':'original',` (if untrigger is the first instance of the job).